transaction/std/src/transaction_builder.rs

```diff
@@ -225,27 +243,26 @@ impl TransactionBuilder {
     }
 }
 
-// This appeases clippy's new_without_default rule.
-impl Default for TransactionBuilder {
-    fn default() -> Self {
-        TransactionBuilder::new()
-    }
-}
-
 /// Creates a TxOut that sends `value` to `recipient`.
 ///
+/// Note: This is only used in test code
+///
 /// # Arguments
 /// * `value` - Value of the output, in picoMOB.
 /// * `recipient` - Recipient's address.
-/// * `ingest_pubkey` - The public key for the recipients fog server, if any
-/// * `rng` -
-fn create_output<RNG: CryptoRng + RngCore>(
+/// * `fog_resolver` - Set of prefetched fog public keys to choose from
+/// * `rng` - Entropy for the encryption.
+///
+/// # Returns
+/// * A transaction output, and the shared secret for this TxOut.
+#[allow(unused)]
```

Comment:
This function should just be nuked, if nobody is using it.

Reply:
It's used in tests, it could move to test module